### ISSUE TITLE
Some GNOME flatpaks left over from Silverblue installer don't work properly in Vauxite

### ISSUE THREAD

**Issue by ali1234** [ublue-os/vauxite#7]
To reproduce, open a terminal and run `xdg-open /usr/share/backgrounds/default.png`

Expected result: the image should open in ristretto, the default image viewer for Xfce, which is in the base image.

Actual result: GNOME Image Viewer opens, but cannot open the file.

![image](https://user-images.githubusercontent.com/651970/217630005-1ba5e3f3-e558-4c27-8828-905d324178f8.png)

Opening other types of files with `xdg-open` seems to work okay, for example the XML file opened in Firefox which had no problem displaying it.

I can go to the Xfce control panel "Default Applications" and change the default for PNG files, like this:

![image](https://user-images.githubusercontent.com/651970/217630589-d26f27c7-7269-492d-b986-40009d35bf22.png)

After doing this, `xdg-open` correctly loads ristretto, which is able to display the file.

Two problems in one here:

1. GNOME Image Viewer shouldn't be installed by default on Xfce. (This will be fixed if/when Vauxite gets its own installer ISO)
2. If it is installed, it should work. (The reason they don't work remains an open question)

Note: `xdg-open` is the CLI equivalent of double clicking a file in a file manager, so the same thing happens if you try to open the image with Thunar.

**Comment by CorpSouth** [comment]
By any chance do you have the flatpak `org.gnome.eog` installed?

Type in `flatpak list | grep gnome` and it should come up with all of the gnome applications.

I got rid of that same flatpak and the command provided returned Ristretto Image Viewer.

**Comment by ali1234** [comment]
Yes, but I do not understand why. I have never installed any flatpaks on this system.

**Comment by CorpSouth** [comment]
> Yes, but I do not understand why. I have never installed any flatpaks on this system.

![image](https://user-images.githubusercontent.com/96737550/217665301-7a3ffd79-bfbb-40c9-99b2-736adc28896f.png)

Example of output.

**Comment by ali1234** [comment]
I mean I don't understand why eog is installed. Presumably it is left over from silverblue default install?

**Comment by CorpSouth** [comment]
> I mean I don't understand why eog is installed. Presumably it is left over from silverblue default install?

That's most likely the cause, rebasing doesn't uninstall flatpaks, and while moving over to Vauxite I was clearing my flatpaks software stack of default GNOME applications installed.

**Comment by hyperreal64** [comment]
Yeah, the flatpaks are left over from the Silverblue install. You can uninstall them manually for now. They are not part of the ostree image, so there is nothing that could be done in the build instructions to remove them. I could probably add a justfile to the image and have a script to run the just actions automatically on the first boot after rebasing.
